UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Models/Response/ExchangeSetErrorResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Newtonsoft.Json;
+
+namespace UKHO.ExchangeSetService.Common.Models.Response
+{
+    public class ExchangeSetErrorResponse : BaseExchangeSetResponse
+    {
+        [JsonProperty("_links")]
```

Comment:
Curious as to why this one has an underscore? (Assume there is a reason)

Reply:
As per the exchange set service API spec we have added underscore in json property.
Reference endpoint :- /productData/productIdentifiers

Reply:
Thanks - Makes sense!